courses-app/src/components/Courses/Courses.tsx

```diff
@@ -35,9 +36,13 @@ const Courses: React.FC = () => {
 		<StyledCourses>
 			<StyledHeadingCourses>
 				<SearchBar onSearch={filteringCoursesHandler} />
-				<Button text={BUTTONS_TEXT.ADD} onClick={addingCourseHandler} />
+				{role === 'admin' && (
```

Comment:
you can save a user role as a Boolean (like `isAdmin` will be true if a user is an admin, then you will be able to do decisions easier and not compare it with string every time)

Reply:
Fix this